app/models/claim.rb

```diff
@@ -0,0 +1,20 @@
+class Claim < ActiveModelSerializers::Model
+  attr_accessor :id
+
+  def self.fetch_all(headers)
+    evss_client = EVSS::ClaimsService.new(headers)
+    keys = %w(openClaims historicalClaims)
+    raw_claims = evss_client.claims.body
+    claims = []
+    keys.each do |key|
+      next unless raw_claims[key]
+      claims << raw_claims[key].map do |raw_claim|
+        attrs = {
+          id: raw_claim["id"]
+        }
+        Claim.new(attrs)
+      end
+    end
+    claims.flatten
```

Comment:
(nitpick) could you possibly use `keys.flat_each do |key|` and avoid having to do `claims.flatten`

or maybe `keys.each_with_object` and avoid having to have the `claim = []`

to my knowledge there is not a `#flat_each_with_object` but maybe I should open a PR to add it in for Enumerable. :)